storage/ndb/src/kernel/blocks/dblqh/DblqhMain.cpp

```diff
@@ -17630,6 +17667,23 @@ void Dblqh::scanLockReleasedLab(Signal* signal,
     the record we didn't want, but now we are returning all found records to
     the API.
     */
+
+    /*
+     * Moz
+     * In pushdown aggregation mode, before we send
+     * scanfragconf to TC, we need to make sure that
+     * there is no agg results cached in aggregation
+     * interpreter.
+     * See more details at Dblqh::scanTupkeyConfLab(),
+     * search for [MOZ-COMMENT] there.
+     */
+    if (scanPtr->m_aggregation) {
+      // TODO (Zhao)
+      // double check: 0 : 1 ?
+      // ndbrequire(scanPtr->m_agg_n_res_recs == 0);
+      g_eventLogger->info("scanPtr->m_agg_n_res_recs: %u",
```

Comment:
Looks to me like this print should be used only in debug mode, see use DEB_* in blocks.